### ISSUE TITLE
Unlisted exception error when compile with '-d:chronicles_log_level=TRACE'(Windows)

### ISSUE THREAD

**Issue by jangko** [status-im/nim-eth#330]
https://github.com/status-im/nim-eth/blob/0700ec770f348017d452e16298f70a37b3caf97b/eth/net/nat.nim#L262

observed in Windows CI

```text
[NimScript] exec: nim c --out:build/nimbus -d:chronicles_log_level=TRACE --verbosity:0 --hints:off -d:release nimbus/nimbus.nim
D:\a\nimbus-eth1\nimbus-eth1\nimbus-eth1\vendor\nim-eth\eth\net\nat.nim(262, 65) template/generic instantiation from here
D:\a\nimbus-eth1\nimbus-eth1\nimbus-eth1\vendor\nim-chronicles\chronicles.nim(303, 18) Error: can raise an unlisted exception: Exception
```

**Comment by kdeme** [comment]
This seems to be a chronicles issue. 
I can reproduce locally in a Windows VM and the error goes away when I remove the only log line in that proc: https://github.com/status-im/nim-eth/blob/master/eth/net/nat.nim#L277

Not sure why it is manifesting now, as no chronicles update was done.

**Comment by jangko** [comment]
fixed by status-im/nim-chronicles#93, closing
